playground/my_playground.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+from pprint import pprint
+import sys
+import sqlite3
+
+def read_data(filename):
+    """(pd.read_csv) uses pandas to read a csv file and convert it to a dataframe
+
+    Args:
+        filename (string): a csv filename 
+    """
+    dataframe = pd.read_csv(filename)
+    print('Printing dataframe...')
+    print(dataframe)
+    return dataframe
+    
+def df_iter_rows(df):
+    """(dataframe.iterrows) uses dataframe.iterrows to print first 3 rows in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    print('(ITER_ROWS) Printing first 3 rows in the dataframe...')
+    for index, row in df.iterrows():
+        if index == 3: break
```

Comment:
```suggestion
def df_iter_rows(df, limit=3):
    """
    Prints the first `limit` rows of a dataframe, leveraging `df.iterrows()`.

    Args:
        df (dataframe): pandas dataframe
        limit (int): number of rows to show [default: 3]
    """
    print(f'Printing first {limit} rows in the dataframe...')
    for index, row in df.iterrows():
        if index == limit: break
```

Reply:
You could do something like above to your functions. 

`limit` is the number of rows, setting the default to `3`

`print(f"my_var is set to {my_var}")` is called `f-string notation` which looks nice in code and allows you to build strings easily.